apps/gadgetron/connection/ConfigConnection.cpp

```diff
@@ -31,131 +31,131 @@ using namespace Gadgetron::Server::Connection::Handlers;
         return std::make_unique<std::stringstream>(decoded);
     }
 #else
-    static std::unique_ptr<std::istream> open_and_verify_config(const std::string& filename)
-    {
-        return std::make_unique<std::ifstream>(filename);
-    }
+static std::unique_ptr<std::istream> open_and_verify_config(const std::string& filename)
+{
+    return std::make_unique<std::ifstream>(filename);
+}
```

Comment:
You have a lot of whitespace changes. They look like they may be conversion from tab to space or vice versa. It may be your editor doing it automatically, but you need to configure your editor to not make whitespace changes, as they make tracking history and viewing real changes extremely difficult. This really needs to be fixed before I can effectively review the changes.